applications/IgaApplication/python_scripts/iga_io_process.py

```diff
@@ -0,0 +1,149 @@
+            CreateVariablesListFromInput(self.params["nodal_results"])
+
+        self.integration_point_results_scalar, self.integration_point_results_vector = \
+            CreateVariablesListFromInput(self.params["integration_point_results"])
+
+        self.frequency = self.params["frequency"].GetDouble()
+
+        self.step_count = 0
+        self.printed_step_count = 0
+        self.next_output = 0.0
+
+    def ExecuteInitialize(self):
+        with open(self.output_file_name, 'w') as file:
+            file.write("Rhino Post Results File 1.0\n") 
+
+    def ExecuteBeforeSolutionLoop(self):
+        pass
+
+    def ExecuteInitializeSolutionStep(self):
+        pass
+
+    def ExecuteFinalizeSolutionStep(self):
+        if this._IsOutputStep:
+            # Print the output
+            time = self.__get_pretty_time(self.model_part.ProcessInfo[TIME])
+            self.printed_step_count += 1
+            self.model_part.ProcessInfo[PRINTED_STEP] = self.printed_step_count
+            if self.output_label_is_time:
+                label = time
+            else:
+                label = self.printed_step_count
+
+            with open(self.output_file_name, 'a') as file:
+                for variable in range(self.nodal_results_scalar.size()):
+                    file.write("Result \"" + variable.Name() + "\" \"Load Case\" " + str(label) + " Scalar OnNodes\n")
+                    file.write("Values\n")
+                    for node in self.model_part.Nodes:
+                        value = node.GetSolutionStepValue(variable, 0)
+                        file.write(str(node.Id) + "  " + str(value) + "\n")
+                    file.write("End Values\n")
+
+                for variable in range(self.nodal_results_vector.size()):
+                    file.write("Result \"" + variable.Name() + "\" \"Load Case\" " + str(label) + " Vector OnNodes\n")
+                    file.write("Values\n")
+                    for node in self.model_part.Nodes:
+                        value = node.GetSolutionStepValue(variable, 0)
+                        file.write(str(node.Id) + "  " + str(value[0]) + "  " +  str(value[1]) + "  " + str(value[2]) + "\n")
+                    file.write("End Values\n")
+
+
+                for variable in range(self.integration_point_results_scalar.size()):
+                    file.write("Result \"" + variable.Name() + "\" \"Load Case\" " + str(label) + " Scalar OnGaussPoints\n")
+                    file.write("Values\n")
+                    for element in self.model_part.Elements:
+                        value = element.Calculate(variable, self.model_part.ProcessInfo)
+                        file.write(str(element.Id) + "  " + str(value) + "\n")
+                    file.write("End Values\n")
+
+
+                for variable in range(self.integration_point_results_vector.size()):
+                    file.write("Result \"" + variable.Name() + "\" \"Load Case\" " + str(label) + " Vector OnGaussPoints\n")
```

Comment:
don't use `file` as name, this is somehow a python-type 
=> use e.g. `output_file`